src/main/java/Duke.java

```diff
@@ -1,10 +1,322 @@
+import java.io.*;
+import java.util.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+
+/**
+ *  Juke is a chatbot that helps users keep track of tasks.
+ *
+ *  The chatbot supports, todos, events and deadlines,
+ *      todos: add to do task to a list
+ *      events: adds task to a list with a specific time for the event with "/at"
+ *      deadlines: adds task to a list with a specific deadline with "/by"
+ *
+ *      The done command followed by an integer x checks off  task x.
+ *      The chatbot supports deletion of tasks with the "delete" command
+ *
+ *      The chatbot will throw exceptions for invalid inputs.
+ * @author branzuelajohn
+ * @version CS2103T AY20/21 Semester 2, Individual Project
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static final String TASK_DONE = "1";
+    public static final String TASK_NOT_DONE = "0";
+
+    public static void main(String[] args) throws Exception {
+
+        // Level 7
```

Comment:
Would it be better to remove such comments or elaborate on them so that they are more understandable?